endaq/device/command_interfaces.py

```diff
@@ -396,7 +366,7 @@ def _sendCommand(self,
 
     def _runSimpleCommand(self,
                           cmd: dict,
-                          statusCode: int = STATUS_RESET_PENDING,
+                          statusCode: int = DeviceStatusCode.RESET_PENDING,
```

Comment:
Correction: `DeviceStatusCode.RESET_PENDING` *is* considered an instance of `DeviceStatusCode`, so the suggestion would work. However, since what's there now works, I want to leave it as-is, rather than make last minute changes. It's a 'private' method, anyway.